pkg/operator/images.go

```diff
@@ -12,12 +12,5 @@ type Images struct {
 
 // DefaultImages returns default set of images for operator.
 func DefaultImages() Images {
```

Comment:
This is called from
```
244 func (optr *Operator) sync(key string) error {                                  
```
do we need to store the images somewhere in the operator to return them there?